src/ncp/example_vendor_hook.cpp

```diff
@@ -34,6 +34,8 @@
 
 #include "ncp_base.hpp"
 
+using namespace ot::Utils;
```

Comment:
Is this necessary? We generally try to discourage `using namespace` (with the exception of `src/core/api`).